waku/v2/node/wakunode2.nim

```diff
@@ -478,7 +470,8 @@ proc mountRelay*(node: WakuNode,
                  topics: seq[string] = newSeq[string](),
                  rlnRelayEnabled = false,
                  relayMessages = true,
-                 triggerSelf = true) {.gcsafe.} =
+                 triggerSelf = true)
+                 {.gcsafe, raises: [Defect, InitializationError, LPError, CatchableError].} = # @TODO: Better error handling: CatchableError is raised by `waitFor`
```

Comment:
This form is a bit hard long and hard to read, is it possible to format at different somehow?

Reply:
I've moved the comment to above the pragma line and the pragma to a single indent inside the proc block [here](https://github.com/status-im/nim-waku/blob/47981cfdf527fed4728462794e5e41d381c2a2b9/waku/v2/node/wakunode2.nim#L478-L479). Let me know if this looks reasonable to you.